src/main/java/se/sundsvall/contract/api/ContractResource.java

```diff
@@ -126,9 +125,8 @@ public ResponseEntity<Contract> getContractById(@Parameter(description = "Kontra
 		}
 	)
 	@PatchMapping(path = "/{id}", produces = ALL_VALUE, consumes = APPLICATION_JSON_VALUE)
```

Comment:
Had a look at [JSON Patch](https://www.baeldung.com/spring-rest-json-patch) for this?